.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,9 +11,11 @@
     - [ ] Screenshot filename is same as value of `name`.
     - [ ] Screenshot size is `800 * 500`.
     - [ ] Screenshot file format is `png`.
+    - [ ] My theme name is unique (not duplicate)
```

Comment:
I think
```
`name` is unique
```
is sufficient

Reply:
Done

Reply:
I meant just need three words, similar in style with
```
[ ] `link` URL is correct.
[ ] `preview` URL is correct.
```

Reply:
You means...

```diff
- - [ ] `link` and URL is correct.
- - [ ] `preview` URL is correct.
+ - [ ] `link` and `preview` URL is correct.
```

Reply:
@YoshinoriN @curbengh 

`link` is the project link of the theme, in most cases is GitHub Repo link, sometimes could be theme's own website (such as https://theme-next.org and https://theme-suka.skk.moe/en/).
`preview` is URL for the demo.

IMHO, we should merge 2 `preview` items instead:

```diff
- - [ ] `preview` URL is correct.
- - [ ] `preview` URL web site is rendered correctly.
+ - [ ] `preview` URL is correct and website is rendered correctly.
```

Reply:
@YoshinoriN 
I mean
```
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
```

---

@SukkaW 
I agree with keeping existing items, I didn't mean removing any. I prefer not to merge to emphasize each point; `and` is not as strong as separate check.

Reply:
@SukkaW 
Thank you for your review and advice :)
But, I have the same opinion @curbengh